homeassistant/components/adguard/const.py

```diff
@@ -9,6 +9,9 @@
 DATA_ADGUARD_VERSION = "adguard_version"
 
 CONF_FORCE = "force"
+CONF_LIST_TYPE = "list_type"
+
+LIST_TYPES = ["blocklist", "allowlist"]
```

Comment:
Can this be a set, in alphabetical order?
```suggestion
LIST_TYPES = {"allowlist", "blocklist"}
```